README.md

```diff
@@ -7,7 +7,7 @@
 [![License][license-src]][license-href]
 [![JSDocs][jsdocs-src]][jsdocs-href]
 
-H3 is a minimal h(ttp) framework built for high performance and portability.
+H3 (pronounced as /eɪtʃθriː/, like h-3) is a minimal h(ttp) framework built for high performance and portability.
```

Comment:
Some packages' READMEs don't have introduction sentence like this: for example, [c12's README](https://github.com/unjs/c12) just says "Smart Configuration Loader.".

So, we might want to get all REAMEs to have an introduction sentence like "c12 (pronounced as /.../, like c-twelve) is a ...".

Reply:
Yes!